frontend/src/components/Footer/Footer.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from "react";
+import {footerItems} from "./FooterItems";
+import "./Footer.css"
+
+class Footer extends Component {
+
+  render() {
+    return (
+      <footer class="FooterItems">
+        <ul className="footer-links">
+          {footerItems.map((item, index) => {
+            return (
+              <li key={index}>
```

Comment:
Again it it good practice to extract mapping to Footer Item, and `return` not required, as map return itself